I18N_MANUAL.md

```diff
@@ -0,0 +1,144 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Apache APISIX Dashboard I18N Manual
+
+Apache APISIX Dashboard use [@umijs/plugin-locale](https://umijs.org/plugins/plugin-locale) solve internationalization issues，in order to make i18n of cleaner and more reasonable, we recommend the following rules for internationalization：
+
+## Location of locale configuration：
+
+- please put **the global locales** under `src/locales`.
+- please put **each page locales** under `src/pages/$PAGE/locales` folder.
+- please put **self-packaged component locales** under `src/components/$COMPONENT/locales` folder. (This part cannot be loaded automatically, it needs to be additionally quoted in the global locales file)
+
+## how to name the locales key：
+
+the key can be like this : [basicModule].[moduleName].[elementName].[...desc]
+
+- basic keys (the first tow levels) divide according to the directory structure: eg: `app.pwa`, `page.consumer`, `component.actionBar`
+
+- sub keys divide according to element +description eg: `app.pwa.message.offline`, `component.actionBar.button.nextStep`
+
+* If the the text is the part of a element, we can use [elementNameProps] eg: `page.consumer.proTableColumns.username`
+* If there are two or more same level part locales of a element, we can add number suffix eg: `page.route.form.itemRulesExtraMessage1.path`, `page.route.form.itemRulesExtraMessage2.path`
+
+- common texts, we should not repeat in other part, and the common locale key omit [elementName] would be better. If the text is used in two or more places inside the module, it`s recommended that this text be shared in the module eg:`page.route.parameterPosition`If the text is used in two or more places between modules, it is recommended that the text be shared globally, and add`global`as the moduleName eg:`component.global.confirm`
```

Comment:
Please take a look at this paragraph:

![image](https://user-images.githubusercontent.com/2106987/90089722-abf25c00-dd54-11ea-9dde-f80517c29b24.png)